### ISSUE TITLE
Days missing in calendar

### ISSUE THREAD

**Issue by beriain** [zoenb/log28#5]
The calendar view displays a wrong calendar if in any other language rather than english. I think it's because it tries to display sunday as the first day of the week. Concretely, you can see in the screenshots that neither in spanish nor in basque appear the days 1 and 2 of march:

![en](https://user-images.githubusercontent.com/7469634/53819806-ef934480-3f6a-11e9-9ed4-5ab6f11b4b44.png)
![es](https://user-images.githubusercontent.com/7469634/53819807-ef934480-3f6a-11e9-9f8e-21a79a78b992.png)
![eu](https://user-images.githubusercontent.com/7469634/53819808-ef934480-3f6a-11e9-8985-d37efc0474ae.png)

**Comment by zoenb** [comment]
I think this is likely an issue with the calendar library log28 uses. https://github.com/RafalManka/ScrollCalendar

Also would you mind opening a pull request with your translations?

**Comment by Asalle** [comment]
3.1.4 will probably fix this, but it seems like this calendar project has not had any releases since 2020, not sure it's actively maintained. Maybe it makes sense to move on to some other calendar?

**Comment by wildeyedskies** [comment]
Yeah the calendar library is pretty much abandoned. It would be nice to swap it out for a newer or better maintained one.

**Comment by Asalle** [comment]
github.com/kizitonwose/Calendar/ proposed in #7 looks good!
